src/clarity-angular/data/datagrid/datagrid-column-toggle-title.spec.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Component} from "@angular/core";
+
+import {DatagridColumnToggleTitle} from "./datagrid-column-toggle-title";
+import {TestContext} from "./helpers.spec";
+
+const textValue = "Custom Title";
+
+export default function(): void {
+    fdescribe("DatagridColumnToggleTitle component", function() {
```

Comment:
It looks like this is a good place for `tslint-defocus` :-)

Reply:
Ha I was testing on this PR and this is why its useful.